pkgs/applications/science/biology/mmseqs2/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, xxd
+, perl
+, enableAvx2 ? stdenv.hostPlatform.avx2Support
+, enableSse4_1 ? stdenv.hostPlatform.sse4_1Support
+, enableMpi ? (false && !stdenv.isDarwin)
+, mpi
+, openmp
+, zlib
+, bzip2
+}:
+
+with lib;
```

Comment:
That's pretty uncommon. Could we have just `lib.optional` where necessary? It's not a big distraction to write it out in full.

Reply:
I followed the nixos/modules' code, but it is very few in pkgs. I will change it.
Thank you!